xsuite/src/world/csproxyBin.ts

```diff
@@ -0,0 +1,63 @@
+import { ChildProcess } from "node:child_process";
+import {
+  getCsproxyBinPath,
+  getCsproxyDefaultConfigPath,
+} from "@xsuite/chainsimulator";
+import { spawnChildProcess } from "./childProcesses";
+
+export const startCsproxyBin = async (
+  debug: boolean = false,
+  configFilePath?: string,
+  nodeOverrideFilePath?: string,
+): Promise<{
+  server: ChildProcess;
+  proxyUrl: string;
+}> => {
+  const csproxyBinPath = getCsproxyBinPath();
+  const csproxyConfigFolder = getCsproxyDefaultConfigPath();
+
+  const server = spawnChildProcess(`${csproxyBinPath}`, [
+    "--server-port",
+    "0",
+    "--config",
+    configFilePath || `${csproxyConfigFolder}/config.toml`,
```

Comment:
`??` instead of `||`. Same for the line below

Reply:
Done